### PR TITLE
Adds task to load app users.

### PR DESCRIPTION
**Pull request by shelleydoljack**
Note that the migration files shared drive now has a new file for app_users.tsv.

### CODE REVIEW

File: spec/users/load_tsv_users_task_spec.rb [hunk 1]
```diff
@@ -5,65 +5,82 @@
...
     end
 
     it 'has a totalRecords size in the hash' do
-      expect(load_tsv_users_task.send(:tsv_user)['totalRecords']).to eq 2
+      expect(tsv_user_data['totalRecords']).to eq 2
     end
   end
 
   context 'when deleting the superceeded tsv data' do
     it 'removes the temprary fields' do
```
Comment (jermnelson) [inline spec/users/load_tsv_users_task_spec.rb]: I know it isn't part of the changes in this PR, but `temprary` is misspelled.

**Comment by shelleydoljack** [comment]
This PR creates an app user using the POST `/users` endpoint. Unfortunately, looking at how boostrap-superuser.pl works, there are user records in related tables that need to be made in order for the user to "work" properly in FOLIO. Need to POST to `/perms/users` and `/service-points-users` as well. See https://github.com/sul-dlss/folio-k8s/blob/a532824e95242b3dbefe5d93c56bf75ec0bc2d3c/docker/bootstrap-superuser/bootstrap-superuser.pl#L68-L126 . It is unclear if we should continue down this road or decide to use `/user-import` endpoint and decide upon a patron group and email address to assign app users.

**Comment by shelleydoljack** [comment]
I already ran the rake task on dev.

**Comment by shelleydoljack** [comment]
Based on #62 I'm not so sure we want to create the user this way.

**Review by jgreben — APPROVED** [review]

**Pull request merged**